src/wish.cpp

```diff
@@ -396,6 +396,8 @@ void game::wishmonster( const tripoint &p )
             if( spawn != tripoint_min ) {
                 std::vector<tripoint> spawn_points = closest_tripoints_first( cb->group, spawn );
                 for( auto spawn_point : spawn_points ) {
+                    // Set id to 0 to assign new one on spawn
+                    mon.set_id( 0 );
```

Comment:
Why is it global? It looks like it's only being used within the reality
bubble.
Why is it applied to the creature itself? This seems like something the
creature tracker could/should be in charge of.